apps/schools/domains/query/components/currentQuery/index.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useEffect, useState } from 'react'
+import { Col, Row, Typography } from 'antd'
+import styles from './styles/styles.module.scss'
+import { Select } from '@domains/common/components/select'
+import queryChat from '@public/image/chatWithParents.svg'
+import Image from 'next/image'
+import { useOrganization } from '@domains/organization/providers/organizationProvider'
+import { useGetAllJoinCircleQueriesQuery } from '@domains/organization/redux/organizationApi'
+import { createSearchTextForRequest } from '@domains/common/utils/searchText'
+import { searchStudentsColumns } from '@domains/query/components/queryList/constants'
+import { getUuidFromUrl } from '@domains/common/utils/getUuidFromUrl'
+import { format } from 'date-fns'
+import ruLocale from 'date-fns/locale/ru'
+import { GetStudentJoinCircle } from '@domains/common/redux/serializers'
+import { useChangeStatusMutation } from '@domains/query/redux/queryApi'
+import { handleQueryStatusChange } from '@domains/query/handlers/queryUpdate'
+
+export const CurrentQuery = () => {
+    const { organizationId } = useOrganization()
+    const uuid = getUuidFromUrl()
+    const { data: queries, isLoading: isQueryLoading } = useGetAllJoinCircleQueriesQuery({
+        circle__organization__id: organizationId,
+        id: uuid[0],
+    })
+
+    const statusTranslations: { [key: string]: string } = {
+        SENT: 'Отправлена',
+        ACCEPTED: 'Принята',
+        IN_PROGRESS: 'На рассмотрении',
+        DECLINED: 'Отклонена',
+        CANCELED: 'Отменена',
+    }
+
+    const translateStatus = (status: string) => {
+        return statusTranslations[status] || status
+    }
+
+    const query = queries?.results[0] as GetStudentJoinCircle
+    const [mutation] = useChangeStatusMutation()
+    const [currentStatus, setCurrentStatus] = useState<string>('')
+    const [currentDependencies, setCurrentDependencies] = useState<string[]>([])
+
+    const graph: { [key: string]: string | string[] } = {
+        Отправлена: ['Принята', 'На рассмотрении', 'Отклонена', 'Отменена'],
+        'На рассмотрении': ['Принята', 'Отклонена'],
+        Принята: 'Принята',
+        Отклонена: 'Отклонена',
+        Отменена: 'Отменена',
+    }
+
+    const handleStatusChange = (value: string) => {
+        let translatedStatus = value
+        for (const key in statusTranslations) {
+            if (statusTranslations[key] === value) {
+                translatedStatus = key
+                break
+            }
+        }
+        handleQueryStatusChange(mutation, uuid[0], translatedStatus)
+        setCurrentStatus(value)
+        const dependencies = graph[value]
+        if (Array.isArray(dependencies)) {
+            setCurrentDependencies(dependencies)
+        } else {
+            setCurrentDependencies([dependencies])
+        }
+    }
```

Comment:
it's must be here, otherwise, you will have to throw there 6 variables that are defined in this nameSpace, but are not defined where I will take out the function